code/Code/style.css

```diff
@@ -0,0 +1,151 @@
+/***ALLGEMEIN***/
+@import url("https://fonts.googleapis.com/css2?family=Abel&display=swap");
+
+body {
+  font-family: "Abel", sans-serif;
+  color: black;
+  margin: 0;
+}
+
+/***NAVIGATION***/
+
+.menu-parent {
+  border: 0px solid transparent;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+}
+
+.menu-logo {
+  width: 250px;
+  height: 80px;
+}
+
+.menu-child a {
+  display: inline-block;
+  padding: 10px 15px;
+  margin: 0 5px;
+  text-decoration: none;
+  background: white;
+  color: rgb(5, 5, 5);
+  border-radius: 4px;
+}
+
+.menu-child a:hover {
+  background: rgb(115, 14, 83);
+  color: white;
+}
+
+/***HERO***/
+
+.hero-picture {
+  height: 450px;
+  position: relative;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: rgb(115, 14, 83);
+}
+
+.h1 {
+  font-size: 30pt;
+}
+
+.h2 {
+  font-size: 15pt;
+  color: black;
+}
+
+.button1 {
+  background-color: rgb(115, 14, 83);
+  color: rgb(245, 236, 242);
+  border: none;
+  height: 40px;
+  width: 200px;
+  font-size: 15pt;
+  border-radius: 4px;
+}
+
+.button1:hover {
+  background-color: white;
+  color: rgb(115, 14, 83);
+}
+
+/***BENEFITS***/
+
+/***ANGEBOT***/
+
+.angebot-titel {
+  padding: 130px;
+  height: 30px;
+  text-align: center;
+  background-color: weiss;
+}
+
+.angebot-titel h1 {
+  font-size: 35pt;
+  color: rgb(115, 14, 83);
+}
+
+.section1 {
+  margin: 0, 10px;
+  display: flex;
+  height: 550px;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(255, 166, 0, 0.218);
+}
+
+.spalte {
+  margin: 60px 30px;
+  padding: 10px 30px;
+  width: 400px;
+  height: 500px;
+  background-color: weiss;
+  text-align: center;
+}
+
+.spalte h2 {
+  font-size: 15pt;
+  color: black;
+}
+
+.button2 {
+  background-color: rgb(115, 14, 83);
+  border: none;
+  color: rgb(245, 236, 242);
+  height: 40px;
+  width: 200px;
+  font-size: 15pt;
+  border-radius: 4px;
+}
+
+.button2:hover {
+  background-color: white;
+  color: rgb(115, 14, 83);
+}
+
+/***ÜBER MICH***/
+
+/***FOOTER***/
+.footer {
+  background-color: rgb(5, 5, 5);
+  text-align: center;
+  margin: 100px solid;
```

Comment:
Since `margin` can't have a `solid` value, this line has no effect, you can remove it completely.